ThesisTemplate.tex

```diff
@@ -327,8 +323,8 @@ \subsection{Other Research on NRHES Models}
+The common uranium nuclear fuel cycle describes the system of mining uranium, converting the uranium to gaseous $UF_6,$ enriching the uranium with higher percentages of U235, fabricating the fuel, the fuel changing in the reactor, storing, reprocessing, and final disposal. There are different fuel cycles, depending on the type of fuel used, the reactors included, and how the spent fuel is managed. Nuclear fuel cycle simulators model the fuel cycle determining the likely quantity and characteristics of nuclear fuel in order to determine future policies surrounding management of the spent fuel.
+This section identifies basic properties of nuclear fuel cycle simulators and assesses their distinct functionality. Figures 2a and 2b display many of the currently used nuclear fuel cycle simulators (NFCS). The fuel cycle simulators are the central nodes in these figures, with the tools for computing some of the basic descriptive qualities of a fuel cycle branching off. The computational attributes that are often included in NFCS are: mass flows, reactor physics, thermal hydraulics, reprocessing, waste management, risk management, and economics. While different fuel cycle simulators were initially optimized for a variety of applications, there is a lot of overlap in their functionality \cite {Guerin2009}. Because nuclear fuel cycle simulators are attempting to model the same thing, the nuclear fuel cycle, the models include some means of tracking the isotopic makeup of the fuels as they move through the fuel cycle. The different fuel cycle simulators generally have the ability to simulate fleets of nuclear reactors with the associated fuel cycle industrial processes and nuclear materials storage. As can be seen in figures 2.2 and 2.3, there are a good number of NFCS, which all have different functionality and dependencies.
```

Comment:
You start with figures 2a and 2b, and then end with 2.2 and 2.3. I would just order them ordinally.

Reply:
The figure numbers are undoubtedly going to change with the latex template as I edit and add figures.  I will plan on fixing them all at the end. Thanks for pointing this out!